### PR TITLE
XSOARLabUpdates Pack

### PR DESCRIPTION
**Pull request by guyfreund**
## Status
- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/etc/issues/32263

## Description
A pack for sending content updates on DFIR channel.

Pack includes:
1. Playbook - Calculate new packs from index.zip and send appropriate message on slack channel
   - Using an XSOAR list to cache last run times.
   - Using `Slack V2` integration.
2. Script - Extract data from index.zip, calculate new packs and build the slack blocks object

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/SendContentUpdates/Scripts/BuildSlackBlocksFromIndex/BuildSlackBlocksFromIndex.py [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+
+
+def main():
+    args = demisto.args()
+    index_folder_path = extract_index(args)
+    new_packs = get_new_packs(args, index_folder_path)
+    last_run = datetime.now().strftime(DATE_FORMAT)
+    blocks = SlackBlocks(new_packs).build()
+    demisto.debug(blocks)
+    return_results_to_context(new_packs, last_run, blocks, args)
```
Comment (teizenman) [inline Packs/SendContentUpdates/Scripts/BuildSlackBlocksFromIndex/BuildSlackBlocksFromIndex.py:255]:
```suggestion
    try:
        args = demisto.args()
        index_folder_path = extract_index(args)
        new_packs = get_new_packs(args, index_folder_path)
        last_run = datetime.now().strftime(DATE_FORMAT)
        blocks = SlackBlocks(new_packs).build()
        demisto.debug(blocks)
        return_results_to_context(new_packs, last_run, blocks, args)
    except Exception as e:
        demisto.error(traceback.format_exc())  # print the traceback
        return_error(f'Failed to execute {demisto.command()} command.\nError:\n{str(e)}')
```

**Review by teizenman — COMMENTED** [review]
Looks really good.

File: Packs/SendContentUpdates/pack_metadata.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Send Content Updates to DFIR",
+    "description": "Send Content Updates to DFIR",
```
Comment (ShahafBenYakir) [inline Packs/SendContentUpdates/pack_metadata.json:3]:
```suggestion
    "name": "New Packs Notifier",
    "description": "Send updates on new released packs to slack channkl",
```

File: Packs/SendContentUpdates/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+### This pack sends content-updates to DFIR Channel.
+---
+#### Flow
+We update on new packs once every week.
+New packs are being retrieved from the public index by the "created" field, along with other fields from the pack marketplace metadata.
+
+We use a XSOAR list (`ContentUpdatesLastRun`) to cache last run times. In each playbook run the last run time is retrieved from the list, and in the end of the run, being set to the updated last run time.
+
+Slack message is being build, showing the 5 "top" packs in full preview and all others as a list.
```
Comment (ShahafBenYakir) [inline Packs/SendContentUpdates/README.md:9]:
```suggestion
Slack message is built, showing the 10 "top" packs in full preview and all others as a list.
```

**Review by ShahafBenYakir — COMMENTED** [review]
- Let's change the pack folder name to `NewPacksNotifier`
- Need to add the job file
-  Let's add docstrings to the main functions

**Comment by guyfreund** [comment]
Adding the `docs-approved` label as this pack is community contributed.

File: Packs/XSOARLabUpdates/README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+### This pack sends content-updates to slack channels.
```
Comment (ShahafBenYakir) [inline Packs/XSOARLabUpdates/README.md:1]: Can slightly twich the format the look like this? https://xsoar.pan.dev/docs/documentation/pack-docs#pack-readme

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**